CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [CHANGE] Querier: returning 422 when query hits `max_fetched_chunks_per_query` and `max_fetched_series_per_query` limits in the store-gateway. #4056
 * [CHANGE] Packaging: Migrate FPM packaging solution to NFPM. Rationalize packages dependencies and add package for all binaries. #3911
+* [ENHANCEMENT] Helm Chart: Enable users to specify additional Kubernetes resource manifests using the `extraObjects` variable.
```

Comment:
Please move to https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md . 

Otherwise I tested it by modifying operations/helm/charts/mimir-distributed/ci/test-oss-values.yaml

Reply:
Thanks! updated the changelog